containerd-shim/main.go

```diff
@@ -100,6 +101,9 @@ func start(log *os.File) error {
 			msgC <- m
 		}
 	}()
+	if runtime.GOOS == "solaris" {
+		return nil
```

Comment:
Returning here will make `containerd` think that the process is dead, is that intended?